libaec.yaml

```diff
@@ -27,9 +27,6 @@ pipeline:
 
   - uses: cmake/install
 
-  - runs: |
-      mv "${{targets.destdir}}"/usr/cmake "${{targets.destdir}}"/usr/lib/cmake
```

Comment:
From the commit message: The files are already moved in a previous build step, and this fails.